hyp3_gamma/__main__.py

```diff
@@ -183,6 +184,13 @@ def water_map():
     parser.add_argument('--hand-fraction', type=float, default=0.8)
     parser.add_argument('--membership-threshold', type=float, default=0.45)
     parser.add_argument('granule')
+
+    parser.add_argument('--include-flood-depth', type=string_is_true, default=False)
+    parser.add_argument('--estimator', type=str, default='nmad', choices=['iterative', 'logstat', 'nmad', 'numpy'])
+    parser.add_argument('--water-level-sigma', type=float, default=3., )
+    parser.add_argument('--known-water-threshold', type=float, default=30.)
+    parser.add_argument('--iterative-bounds', type=float, default=[0, 15])
```

Comment:
Maybe the type should be `tuple` and the default value should be a tuple instead of a list?